src/templates/dandelion/config/screens/LockScreen.js

```diff
@@ -0,0 +1,338 @@
+                label={appLabel}
+              />
+            </span>
+            settings below.
+          </span>
+        }
+      />
+
+      <div
+        css={`
+          ${fieldsLayout};
+        `}
+      >
+        <Field
+          label={
+            <React.Fragment>
+              Token address
+              <Help hint="What’s the token address?">
+                <strong>Token address</strong> is the address of the token to be
+                locked.
+              </Help>
+            </React.Fragment>
+          }
+        >
+          <TokenSelector
+            selectedIndex={lockToken.selectedIndex}
+            onChange={handleTokenChange}
+            value={lockToken.data.address}
+            tokens={[DEFAULT_LOCK_TOKEN]}
+          />
+        </Field>
+
+        <Field
+          label={
+            <React.Fragment>
+              Lock amount
+              <Help hint="What’s the lock amount?">
+                <strong>Lock amount</strong> is the amount of tokens to lock.
+              </Help>
+            </React.Fragment>
+          }
+        >
+          <TextInput
+            onChange={handleLockAmountChange}
+            value={lockAmount === -1 ? '' : lockAmount}
+            wide
+          />
+        </Field>
+      </div>
+
+      <Duration
+        duration={lockDuration}
+        onUpdate={handleDurationChange}
+        label={
+          <React.Fragment>
+            {DURATION_LABEL}
+            <Help hint="What’s the lock duration?">{DURATION_HELP_TEXT}</Help>
+          </React.Fragment>
+        }
+      />
+
+      <Field
+        label={
+          <React.Fragment>
+            Spam penalty %
+            <Help hint="What’s the spam penalty?">
+              <strong>Spam penalty</strong>
+              Is a % of the base lock amount and lock duration. The more active
+              locks an account has, the more tokens will have to lock and for a
+              longer period of time for subsequent proposals.
+            </Help>
+          </React.Fragment>
+        }
+      >
+        <TextInput
+          type="number"
+          step={1}
+          onChange={handleSpamPenaltyChange}
+          value={spamPenalty}
+        />
+        <span> %</span>
+      </Field>
+
+      {formError && (
+        <Info
+          mode="error"
+          css={`
+            margin-bottom: ${3 * GU}px;
+          `}
+        >
+          {formError}
+        </Info>
+      )}
+
+      <Info
+        css={`
+          margin-bottom: ${3 * GU}px;
+        `}
+      >
+        These settings will determine the token to be locked, how long tokens
+        are locked, the amount of tokens to lock and a penalty percentage for
+        spamming proposals. At initialization the token address parameter can be
+        set to an ERC20 token. It cannot be changed. If a change is necessary
+        the user can install a new instance and change permissions in the
+        organization to reflect the change.
```

Comment:
I'd be keen to help rewrite this to make it clearer. I think the big missing piece is when/why we're locking tokens (when we create proposals?).

Reply:
At one point we had a similar info box above the configuration with the following text:

> The Time Lock app is used to discourage spammy, low-quality proposal submission by requiring users to lock tokens for a period of time whenever they submit a proposal. The amount and duration of each lock increases the more simultaneously active locks the user has accrued.

The intent was as you describe to explain the when and why before the user is asked to pick values. I think its useful for that to be above the input fields. 

The text at the bottom feels a bit redundant given the field hints. Perhaps we can remove it entirely? The part about the token address not fixed and requiring installing a new app to change can be moved to the apps documentation, or added to the field hint.

Reply:
> The intent was as you describe to explain the when and why before the user is asked to pick values. I think its useful for that to be above the input fields.

Sounds good. I don't think the other templates really do this, but it's more important to be clear than completely consistent here.

> The text at the bottom feels a bit redundant given the field hints.

Agree; I might just add to the info "This value cannot be easily modified, so pick wisely!" or etc.